src/components/archive/ArchiveList.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { Component, ReactNode } from 'react';
+import axios from 'axios';
+import DashboardListCard from '../dashboard/dashboard-list/DashboardListCard';
+import ApplicantManagementModal from '../modals/ApplicantManagementModal';
+import './ArchiveList.scss';
+
+type ArchiveListProps = {
+    mode: string;
+    viewApplicant: (newCount: number) => void;
+    accepted: any[];
+    rejected: any[];
+};
+
+type ArchiveListState = {
+    showModal: boolean;
+    name: string;
+    role: string;
+    type: string;
+    status: string;
+    email: string;
+    screeningGrade?: number;
+    interviewGrade?: number;
+};
+
+class ArchiveList extends Component<ArchiveListProps, ArchiveListState> {
+    constructor(props: any) {
+        super(props);
+        this.state = {
+            showModal: false,
+            name: '',
+            role: '',
+            type: '',
+            status: 'Pending',
+            email: '',
+        };
+    }
+
+    showModal = (
+        name: string,
+        role: string,
+        type: string,
+        status: string,
+        email: string,
+        screeningGrade: number,
+        interviewGrade: number,
+    ): void => {
+        this.setState({
+            showModal: true,
+            name: name,
+            role: role,
+            type: type,
+            status: status,
+            email: email,
+            screeningGrade: screeningGrade,
+            interviewGrade: interviewGrade,
+        });
+    };
+
+    closeModal = (type: string, email: string): void => {
+        if (type.includes('Email')) {
+            axios({
+                method: 'post',
+                url: `http://localhost:4000/email`,
+                data: {
+                    recipient: email,
+                    action: type,
+                },
+            })
+                .then(() => {
+                    console.log('Mail sent successfully!');
+                })
+                .catch((err) => {
+                    console.log(err);
+                });
+        }
+        this.setState({ showModal: false });
+    };
```

Comment:
From Archive I don't think we need to send an email - probably makes more sense to pass as few empty functions as possible to prevent unexpected results.

Reply:
Yeah, I agree. I'll remove that.